### ISSUE TITLE
[BUG] test fixtures do not seem to be published

### ISSUE THREAD

**Issue by Wrywulf** [slackhq/EitherNet#44]
### Describe the bug
I tried to add the test fixtures to my gradle project:

```
testImplementation testFixtures("com.slack.eithernet:eithernet:1.2.0")
```

But my test sources don't seem to pick up the test-fixture dependencies.

When I run `dependencyInsight` i get:

```
:dependencyInsight
com.slack.eithernet:eithernet:1.2.0 FAILED
   Failures:
      - Could not resolve com.slack.eithernet:eithernet:1.2.0.
          - Unable to find a variant of com.slack.eithernet:eithernet:1.2.0 providing the requested capability com.slack.eithernet:eithernet-test-fixtures:
               - Variant apiElements provides com.slack.eithernet:eithernet:1.2.0
               - Variant runtimeElements provides com.slack.eithernet:eithernet:1.2.0
               - Variant testFixturesApiElements provides com.slack.eithernet:EitherNet-test-fixtures:1.2.0
               - Variant testFixturesRuntimeElements provides com.slack.eithernet:EitherNet-test-fixtures:1.2.0


```
Which, according to [gradle docs](https://docs.gradle.org/current/userguide/java_testing.html) seem to be cause by missing gradle module metadata.

I'm no gradle expert, so perhaps I'm doing something wrong.

### Requirements (place an `x` in each of the `[ ]`)**
* [x] I've read and understood the [Contributing guidelines](../CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

#### Reproducible in:

{project_name} version: 1.2.0

**Comment by ZacSweers** [comment]
Yes I've seen issues with this as well but I don't understand what's wrong. The artifacts themselves are published and the module metadata looks right _enough_ to me? But honestly this is so opaque and non-obvious so I'm not sure really want to do. It's a pretty vanilla project setup for it.

For now, I've been able to work around it in our repo like so

```kotlin
api("com.slack.eithernet:eithernet:$eithernetVersion:test-fixtures")
```

**Comment by Wrywulf** [comment]
Thanks for the help (and for the lib)! The workaround is good enough for me atm.

But yeah, it does seem quite odd. I haven't used test fixtures before and I'm not best pals with the gradle docs to begin with, but running into this issue confused me.. even more :)

**Comment by ZacSweers** [comment]
This is resolved in 1.2.1. Turns out test-fixtures use the _project name_ for their artifact ID prefix 🤔, so I needed to explicitly set the project name rather than allow it to use whatever the repo clone dir name happened to be.
